tests/src/integration/objects/cluster.hpp

```diff
@@ -206,22 +193,6 @@ class Cluster : public Object<CassCluster, cass_cluster_free> {
     return *this;
   }
 
-  /**
-   * Enable/Disable the use of hostname resolution
-   *
-   * This is useful for authentication (Kerberos) or encryption (SSL)
-   * services that require a valid hostname for verification.
-   *
-   * @param enable True if hostname resolution should be enabled; false
-   *               otherwise (default: true)
-   * @return Cluster object
-   */
-  Cluster& with_hostname_resolution(bool enable = true) {
-    EXPECT_EQ(CASS_OK, cass_cluster_set_use_hostname_resolution(
-                           get(), (enable == true ? cass_true : cass_false)));
-    return *this;
-  }
-
```

Comment:
The heart of the matter.  The use of this function was the immediate cause of the slew of deprecated warnings when building the integration tests.